config/local.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 func init() {
-	Register(User, "local", "username", "provide a local username",
+	Register(User, "local", "username", "provide a local username, e.g. '-user=local,olgabot@localhost'",
```

Comment:
This isn't actually the string shown in the flag: it's shown in the help text printed by `reflow config -help`.

```
$ reflow -help
...
  -user string
    	override user from config; see reflow config -help
```

If you look at `reflow config -help`:
```
$ reflow config -help
...
user: local,username
	provide a local username
```

Which I think makes sense in context...